ovs/dal/hybrids/albabackend.py

```diff
@@ -257,13 +257,14 @@ def _osd_statistics(self):
         try:
             config = Configuration.get_configuration_path(self.abm_cluster.config_location)
             # TODO: This will need to be changed to osd-multistatistics, see openvstorage/alba#749
-            raw_statistics = AlbaCLI.run(command='asd-multistatistics', config=config, named_params={'long-id': ','.join(osd_ids)})
+            raw_statistics = AlbaCLI.run(command='asd-multistatistics', config=config, extra_params=['--all'])
         except RuntimeError:
             return statistics
         if raw_statistics:
             for osd_id, stats in raw_statistics.iteritems():
-                if stats['success'] is True:
-                    statistics[osd_id] = stats['result']
+                if osd_id in osd_ids:
```

Comment:
`osd_id in osd_ids` is a superfluous test

Reply:
Its to avoid having non-framework osds coming in the results I suppose?